app/main.py

```diff
@@ -1,13 +1,21 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+
+    return {"line": error["line_number"],
```

Comment:
After opening the parentheses, write all the arguments on a new line, including the first one